module-apps/application-meditation/include/application-meditation/ApplicationMeditation.hpp

```diff
@@ -27,7 +27,7 @@ namespace app
 
         void createUserInterface() override;
         void destroyUserInterface() override;
-        std::unique_ptr<gui::OptionsData> state;
+        std::unique_ptr<gui::OptionsData> state = nullptr;
```

Comment:
You do not need to explicitly initialise std::unique_ptr. Its default constructor will set it to nullptr :)

Reply:
done